src/frontends/onnx/tests/op_extension.cpp

```diff
@@ -141,4 +143,17 @@ INSTANTIATE_TEST_SUITE_P(ONNXOpExtensionViaONNXConstructor,
 INSTANTIATE_TEST_SUITE_P(ONNXOpExtensionViaCommonConstructor,
                          FrontEndOpExtensionTest,
                          ::testing::Values(getTestDataOpExtensionViaCommonConstructor()),
-                         FrontEndOpExtensionTest::getTestCaseName);
\ No newline at end of file
+                         FrontEndOpExtensionTest::getTestCaseName);
+
+TEST(ONNXOpExtensionViaCommonConstructor, onnx_op_extension_with_custom_domain) {
+    const auto ext = std::make_shared<onnx::OpExtension<ov::op::v0::Relu>>("CustomRelu", "my_custom_domain");
+
+    auto fe = std::make_shared<ov::frontend::onnx::FrontEnd>();
+    fe->add_extension(ext);
+
+    const auto input_model = fe->load(CommonTestUtils::getModelFromTestModelZoo(
+        ov::util::path_join({TEST_ONNX_MODELS_DIRNAME, "relu_custom_domain.onnx"})));
+
+    std::shared_ptr<ov::Model> model;
+    fe->convert(input_model);
```

Comment:
What's the result of passing test? I presume there should be sth like EXPECT_NO_THROW or ASSERT_NO_THROW.

Reply:
@t-jankowski right, I've removed `EXPECT_NO_THROW` during debuging, thanks